includes/fbgraph.php

```diff
@@ -407,7 +407,11 @@ public function update_product_set_item( $product_set_id, $data ) {
 		}
 
 		public function delete_product_set_item( $product_set_id ) {
-			$url = $this->build_url( $product_set_id, '', '8.0' );
+
+			$params = ( true === apply_filters( 'wc_facebook_commerce_allow_live_product_set_deletion', $product_set_id ) ) ? '?allow_live_product_set_deletion=true' : '';
```

Comment:
Hey @sc-ecolasurdo , shouldn't the filter be like this? This way we default it to true, and avoid the need for a `__return_true` filter at the top.

```suggestion
			$params = ( true === apply_filters( 'wc_facebook_commerce_allow_live_product_set_deletion', true, $product_set_id ) ) ? '?allow_live_product_set_deletion=true' : '';
```